### PR TITLE
frankenphp-symfony: perf improvement

### PR DESCRIPTION
**Pull request by dunglas**
Backport of some micro-optimizations by @francislavoie https://github.com/laravel/octane/pull/764#discussion_r1375165179

### CODE REVIEW

File: src/frankenphp-symfony/src/Runner.php [hunk 1]
```diff
@@ -22,22 +22,25 @@ public function __construct(private HttpKernelInterface $kernel)
...
+        $kernel = $this->kernel;
         $server = array_filter($_SERVER, static fn (string $key) => !str_starts_with($key, 'HTTP_'), ARRAY_FILTER_USE_KEY);
         $server['APP_RUNTIME_MODE'] = 'web=1&worker=1';
 
-        do {
-            $ret = \frankenphp_handle_request(function () use ($server, &$sfRequest, &$sfResponse): void {
-                // Merge the environment variables coming from DotEnv with the ones tight to the current request
-                $_SERVER += $server;
+        $handler = static function () use ($kernel, $server, &$sfRequest, &$sfResponse): void {
+            // Merge the environment variables coming from DotEnv with the ones tight to the current request
```
Comment (francislavoie) [inline src/frankenphp-symfony/src/Runner.php:30]: Typo? Not sure "tight" made sense here :thinking: 

```suggestion
            // Merge the environment variables coming from DotEnv with the ones tied to the current request
```

File: src/frankenphp-symfony/src/Runner.php [hunk 2]
```diff
@@ -22,22 +22,25 @@ public function __construct(private HttpKernelInterface $kernel)
 
     public function run(): int
     {
+        $kernel = $this->kernel;
```
Comment (francislavoie) [inline src/frankenphp-symfony/src/Runner.php:25]: Hmm, `static function` might not be worth it here since it has a reason to use `$this`.

Comment (dunglas) [reply src/frankenphp-symfony/src/Runner.php:25]: Yes I was hesitating but let's keep this simple indeed.

**Comment by alexander-schranz** [comment]
thx @dunglas

**Pull request merged**